images/requirements.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Image Requirements
+description: Learn about the minimum requirements for Coder images
+---
+
+Coder inserts assets into each workspace to enable code-server, JetBrains IDEs,
+and the Coder CLI. These assets are installed into the `/var/tmp/coder` directory
+of each workspace.
+
+The minimum GNU libc version supported for such binaries is 2.1.
```

Comment:
Can this be part of the page where we describe how to make custom images? https://coder.com/docs/images/structure

I think we should avoid very small pages like this because it's more work to find maintain. This one also wouldn't be accessible from the menu since you didn't modify the JSON manifest